submissions/nadsatt/movie-seat-booking-js/index.js

```diff
@@ -0,0 +1,111 @@
+(() => {
+  const FORM_INPUTS = document.querySelectorAll(
+    'input:not([name="direction"])'
+  );
```

Comment:
With this approach, you need to do extra manipulation to find needed inputs. Better use an independent variable for each type of inputs and work with it.